### DISCUSSION TITLE
controling spike hub via bluetooth (python and windows)

### DISCUSSION THREAD

**Discussion by NivSiton** [pybricks/support#1155] [Q&A]
Hello, 
I want to control the spike prime hub through my python editor. 
I've established a connection with bleak libary, is it possible to send commands to the hub after the connection is formed with pybricks/pybricksdev?
or is there another way?

Thanks!

**Comment by dlech** [comment]
Have you seen <https://pybricks.com/projects/tutorials/wireless/hub-to-device/pc-communication>?

If you are using the beta firmware, the code needs to be updated as discussed in https://github.com/orgs/pybricks/discussions/997
